components/datagrid/HeaderRowOverlay/theme.css

```diff
@@ -0,0 +1,16 @@
+@import '@teamleader/ui-colors';
+@import '@teamleader/ui-typography';
+
+.header-row-overlap {
+  background: var(--color-neutral-light);
+  position: absolute;
+  left: 48px;
+  right: 0;
```

Comment:
I think `right: 0;` is redundant if you specify `left`, not 100% sure, but should be the same result.

Reply:
Ok, TIL then 👍